cli/features/tutorial.feature

```diff
@@ -134,7 +134,9 @@ Feature: Following the tutorial
           - 'mongo'
       """
     When running "exo setup" in this application's directory
+    And setting up a temporary mongo docker container
```

Comment:
#71 is already a PR, so let's try to get this in. I'd like to avoid the detour with subsequent need to remove this if possible